biz.aQute.bndlib/src/aQute/bnd/http/HttpClient.java

```diff
@@ -72,6 +72,7 @@
 	private URLCache							cache					= new URLCache(IO.getFile("~/.bnd/urlcache"));
 	private Registry							registry				= null;
 	private Reporter							reporter				= new Slf4jReporter(HttpClient.class);
+	private boolean								offline;
```

Comment:
Should this be volatile since it can be set on threads other than the thread reading it?